app/assets/stylesheets/courses.css.scss

```diff
@@ -0,0 +1,52 @@
+// import the CSS framework
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+.index-course-box {
+    @extend .thumbnail;
+    @extend .col-md-4;
+    height: $line-height-computed * 16;
+}
+
+.index-course-content {
+    @extend .caption;
+}
+
+.index-course-description {
+    margin-bottom: 15px;
+}
+
+.took-this {
+    @extend .button-primary;
+    margin: 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered bottom, margin, position